ts/Core/Chart/StockChart.ts

```diff
@@ -368,7 +368,7 @@ addEvent(Chart, 'update', function (
     // case (#6615)
     if ('scrollbar' in options && chart.navigator) {
         merge(true, chart.options.scrollbar, options.scrollbar);
-        chart.navigator.update({});
+        chart.navigator.update({ enabled: !!chart.navigator.navigatorEnabled });
```

Comment:
Q: Just to be on the same page: is it expected to force destroy() + init()?

Reply:
Unfortunately yes, until we separate the scrollbar from navigator it has to work like that (fallback to the old original solution of destroy/init).

Reply:
Okay, thanks for the explanation!